Musicca/app/src/main/java/com/example/musicca/activities/CurrentPlaylistActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.example.musicca.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.example.musicca.R;
+import com.example.musicca.adapters.CurrentPlaylistAdapter;
+import com.example.musicca.adapters.QueueAdapter;
+import com.example.musicca.models.Playlist;
+import com.example.musicca.models.Song;
+import com.parse.GetCallback;
+import com.parse.ParseQuery;
+
+import org.w3c.dom.Text;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CurrentPlaylistActivity extends AppCompatActivity {
+
+    private static final String TAG = "CurrentPlaylistActivity";
+
+    private TextView tvPlaylistTitle;
+    private RecyclerView rvPlaylistSongs;
+    private Button btnAddMoreSongs;
+    private String playlistObjectId;
+    private CurrentPlaylistAdapter currentPlaylistAdapter;
+    protected List<Song> songsInPlaylist;
+    private Playlist currentPlaylist;
+
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_current_playlist);
+
+        tvPlaylistTitle = findViewById(R.id.tvPlaylistTitle);
+        rvPlaylistSongs = findViewById(R.id.rvPlaylistSongs);
+        btnAddMoreSongs = findViewById(R.id.btnAddMoreSongs);
+        playlistObjectId = getIntent().getStringExtra("playlistobjectid");
```

Comment:
Please make the extra key a constant in the class that make the most sense (maybe the class that sends the intent?)
Here is the Android documentation for intent, with some advice on how to name the constant (see the Extras sections): https://developer.android.com/guide/components/intents-filters#Building.

The example goes: `static final String EXTRA_GIGAWATTS = "com.example.EXTRA_GIGAWATTS";`